File: 1013-fibonacci-number/fibonacci-number.java
class Solution {
    public int fib(int n) {
        int i,t1=0,t2=1,res=0;
        if(n==0){
            return t1;}
        if(n==1){
            return t2;}
        for(i=2;i<=n;i++){
            res=t1+t2;
            t1=t2;
            t2=res;
        }
        return res;
    }
}

File: 1013-fibonacci-number/fibonacci-number.py
class Solution(object):
    def fib(self, n):
        """
        :type n: int
        :rtype: int
        """
        fibo=[0,1]
        for i in range(2,n+1):
            fibo.append(fibo[i-1]+fibo[i-2])
        return fibo[n]